values.yaml

```
@@ -1119,6 +1119,9 @@ csi:
   # generating secret versions.
   hmacSecretName: ""
 
+  # Allow modification of the hostNetwork parameter to avoid the need of a dedicated pod ip
```

Just some line wrapping.

```suggestion
  # Allow modification of the hostNetwork parameter to avoid the need of a
  # dedicated pod ip
```